packages/flutter_tools/lib/runner.dart

```diff
@@ -111,10 +111,21 @@ Future<int> run(
 
           // TODO(eliasyishak): Set the telemetry for the unified_analytics
           //  package as well, the above will be removed once we have
-          //  fully transitioned to using the new package
+          //  fully transitioned to using the new package, https://github.com/flutter/flutter/issues/128251
           await globals.analytics.setTelemetry(true);
         }
 
+        // Ensure that anyone opted out of package:unified_analytics is also
+        // opted out of legacy analytics and that analytics is not being suppressed
```

Comment:
looks like we missed updating this comment in code review. Can you update this here (and in a follow-up on master) to be accurate?

Reply:
good catch!